src/SixLabors.Fonts/FontCollection.cs

```diff
@@ -16,11 +16,37 @@ namespace SixLabors.Fonts
     /// </summary>
     public sealed class FontCollection : IFontCollection, IFontMetricsCollection
     {
+        private readonly HashSet<string> searchDirectories;
         private readonly HashSet<FontMetrics> metricsCollection = new();
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FontCollection"/> class.
+        /// </summary>
+        public FontCollection()
+            : this(Array.Empty<string>())
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FontCollection"/> class.
+        /// </summary>
+        /// <param name="searchDirectories">The collection of directories used to search for font families.</param>
+        /// <remarks>
+        /// Use this constructor instead of the parameterless constructor if the fonts added to that collection
+        /// are actually added after searching inside physical file system directories. The message of the
+        /// <see cref="FontFamilyNotFoundException"/> will include the searched directories.
+        /// </remarks>
+        public FontCollection(IReadOnlyCollection<string> searchDirectories)
```

Comment:
Making this constructor public and adding the public property does not make sense to me. We can add Fonts via explicit path and via a stream, neither of which methods are affected by this property.

Reply:
Fixed. I split this out to a separate interface.